pgml-extension/src/lib.rs

```diff
@@ -21,9 +21,20 @@ pg_module_magic!();
 
 extension_sql_file!("../sql/schema.sql", name = "schema");
 
+extern "C" {
```

Comment:
Can we move this `extern fn` into the `config.rs` module? It could even go in `config::initialize_server_params` because it isn't/shouldn't be used anywhere else.